portfolio/src/main/java/com/google/sps/data/PortfolioComments.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+
+/**
+* Class containing list of Comments for Jalyn's Portfolio 
+*/
+public final class PortfolioComments {
+
+  private final ArrayList<String> masterCommentList = new ArrayList<>();
```

Comment:
field comment.

Reply:
Also, the field type should be List<String> to be generic.

Reply:
I deleted the PortfolioComments class